modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx

```diff
@@ -182,6 +182,13 @@ export default function EditNotificationTemplate({
 			errors.name = Liferay.Language.get('required');
 		}
 
+		if (
```

Comment:
@alinedoleron to avoid repeating the notification type check, could you please rearrange the `if` statements like this 

              if (notificationTemplateType === 'email') {
			if (!values.recipients[0].from) {
				errors.from = Liferay.Language.get('required');
			}

			if (!values.recipients[0].fromName[defaultLanguageId]) {
				errors.fromName = Liferay.Language.get('required');
			}

			if (!values.recipients[0].to[defaultLanguageId]) {
				errors.to = Liferay.Language.get('required');
			}
		   }
		
Thanks :)